WordPress/src/main/java/org/wordpress/android/ui/mysite/jetpackbadge/JetpackAppInstallReceiver.kt

```diff
@@ -0,0 +1,53 @@
+package org.wordpress.android.ui.mysite.jetpackbadge
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.util.Log
+import android.widget.Toast
+import androidx.preference.PreferenceManager
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.push.GCMMessageHandler
+import org.wordpress.android.util.BuildConfigWrapper
+import javax.inject.Inject
+
+class JetpackAppInstallReceiver : BroadcastReceiver() {
+    @Inject lateinit var mBuildConfigWrapper: BuildConfigWrapper
+    @Inject lateinit var gcmMessageHandler: GCMMessageHandler
+
+    override fun onReceive(context: Context?, intent: Intent?) {
+        (context?.applicationContext as WordPress).component().inject(this)
```

Comment:
We can remove this `inject` line and add `@AndroidEntryPoint` to the top of the class.

Reply:
Yes, Updated (Couldn't use constructor injection for now.  It wouldn't work with adb commands)

Reply:
> I couldn't receive push notifications for WP, even when the switch was on

It's working for me.  Choose Application(WordPress Android), Username (Yours), and Device (All Devices) if you're using [Debug Console](https://mc.a8c.com/mobile/push-notifications/debug/) to test

Reply:
I changed the variant to `jalapeno` and it worked. 👍🏻